lib/rubocop/cop/chef/deprecation/deprecated_mixins.rb

```diff
@@ -51,24 +51,29 @@ class UsesDeprecatedMixins < Cop
             (send nil? :require ( str {"chef/mixin/language" "chef/mixin/language_include_attribute" "chef/mixin/language_include_recipe"}))
           PATTERN
 
+          extend AutoCorrector
           def on_send(node)
             deprecated_mixin?(node) do
-              add_offense(node, location: :expression, message: MSG, severity: :warning)
+              add_offense(node, message: MSG, severity: :warning) do |corrector|
+                fixer(node, corrector)
```

Comment:
We should be able to call the individual autocorrects here as well

Reply:
Should be all fixed up.